### PR TITLE
add step in setup file, fix methods in coffeeShopOrder class

### PR DESCRIPTION
**Pull request by aqsa505**

### CODE REVIEW

File: coffee-shop-kata-solutions/src/main/java/bnymellon/codekatas/coffeeshopkata/CoffeeShopOrder.java [hunk 1]
```diff
@@ -86,27 +86,29 @@ public List<String> getFoodItemsForOrder()
     public String generateReceipt()
```
Comment (emilie-robichaud) [inline coffee-shop-kata-solutions/src/main/java/bnymellon/codekatas/coffeeshopkata/CoffeeShopOrder.java:86]: In the method description, you should specify "Generate a receipt for a customer's food items." The list it's iterating through contains both food and beverage items, yet only food is being covered here - that needs to be specified in the method description, separate from the tests. Also consider renaming the method, "generateFoodItemsReceipt" or similar. Same comment for the two other CoffeeShopOrder tests - this should be consistent in all 3 classes.

Comment (emilie-robichaud) [inline coffee-shop-kata-solutions/src/main/java/bnymellon/codekatas/coffeeshopkata/CoffeeShopOrder.java:86]: In the method description, you should specify "Generate a receipt for a customer's food items." The list it's iterating through contains both food and beverage items, yet only food is being covered here - that needs to be specified in the method description, separate from the tests. Also consider renaming the method, "generateFoodItemsReceipt" or similar. Same comment for the two other CoffeeShopOrder tests - this should be consistent in all 3 classes.

File: coffee-shop-kata-solutions/src/test/java/bnymellon/codekata/coffeeshopkata/CoffeeShopTest.java [hunk 2]
```diff
@@ -58,12 +58,20 @@ public void setUp()
     }
```
Comment (emilie-robichaud) [inline coffee-shop-kata-solutions/src/test/java/bnymellon/codekata/coffeeshopkata/CoffeeShopTest.java:58]: Add Americano to the order, to have a variety of items included - all tests should still pass. Same comment for the other two CoffeeShopTest classes, it should be consistent across all 3 classes.

**Review by emilie-robichaud — CHANGES_REQUESTED** [review]

File: coffee-shop-kata-solutions/src/test/java/bnymellon/codekata/coffeeshopkata/CoffeeShopTest.java (hunk 2 above)
Comment (aqsa505) [reply coffee-shop-kata-solutions/src/test/java/bnymellon/codekata/coffeeshopkata/CoffeeShopTest.java:58]: Americano is not a record class so i don't want them to confuse with how to deal with that.

File: coffee-shop-kata-solutions/src/main/java/bnymellon/codekatas/coffeeshopkata/CoffeeShopOrder.java (hunk 1 above)
Comment (aqsa505) [reply coffee-shop-kata-solutions/src/main/java/bnymellon/codekatas/coffeeshopkata/CoffeeShopOrder.java:86]: make sense

File: coffee-shop-kata-solutions/src/test/java/bnymellon/codekata/coffeeshopkata/CoffeeShopTest.java (hunk 2 above)
Comment (emilie-robichaud) [reply coffee-shop-kata-solutions/src/test/java/bnymellon/codekata/coffeeshopkata/CoffeeShopTest.java:58]: It shouldn't have an impact on any of the tests here - the food items for order and receipt methods explicitly say to only cover the food case. A list of order items can have any drinks and any foods.

Comment (emilie-robichaud) [reply coffee-shop-kata-solutions/src/test/java/bnymellon/codekata/coffeeshopkata/CoffeeShopTest.java:58]: Like you should be able to add Americano to the order in setUp() and all of the tests will pass without being changed.

File: coffee-shop-kata/SETUP.md [hunk 3]
```diff
@@ -8,12 +8,18 @@
 ### Project setup
 1. Git clone the entire [code-katas](https://github.com/BNYMellon/CodeKatas) project from GitHub or download the project
    as a .zip file.
-2. Launch the project in the IDE as a maven project. You can find instructions on how to do
+2. Launch the project in the IDE and point to the pom.xml to be opened as a project.
+   You can find more instructions on how to do
    that [here](https://www.jetbrains.com/idea/guide/tutorials/working-with-maven/importing-a-project/).
-3. To verify that the Java 8 module is set up correctly,
+3. To use Java 20 preview features in IntelliJ IDEA, follow the following steps:
```
Comment (prathasirisha) [inline coffee-shop-kata/SETUP.md:14]: "follow these steps" instead of"follow the following steps" too verbose.

**Review by emilie-robichaud — APPROVED** [review]

**Review by prathasirisha — APPROVED** [review]

**Pull request merged**